src/scss/components/_select.scss

```diff
@@ -85,6 +92,8 @@
   position: relative;
   z-index: 1;
   box-sizing: initial;
+  padding-top: 15px;
+  padding-bottom: 15px;
```

Comment:
You can make both padding declarations in a single line

Reply:
Yep, you can simplify with `padding: 15px 0;`